File: College/大一/c++/学校/teacher/7.2.cpp
#include <iostream>

using namespace std;
int max(int m,int n)
{
    return (m>n?m:n);
}
int min(int m,int n)
{
    return (m<n?m:n);
}
int main()
{
    int m,n;
    cin>>m>>n;
    cout<<max(m,n)<<'\t'<<min(m,n)<<endl;
    return 0;
}

File: College/大一/python/7/7.2.py
lst=[12,2,16,30,28,10,16,20,6,18]
print("个数：",len(lst))
lst.sort(reverse=True)
print("min:",lst[9])
print("max:",lst[0])
print("new:",lst,sep='\n')
u=0
for i in lst:
    u=u+i
print("average:",u/10)